index.html

```diff
@@ -23,16 +23,25 @@
 
 <body>
   <header>
-    <img src="./images/SideImage.svg" id="nav-image" alt="" />
-    <nav>
-      <span class="material-symbols-outlined"> menu </span>
-    </nav>
+    <div>
+      <nav id="desktop-nav">
+        <ul>
+          <li id="active">Help</li>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
 
+      <nav id="mobile-nav">
+        <span class="material-symbols-outlined"> menu </span>
+      </nav>
+    </div>
   </header>
```

Comment:
- [ ] Your `nav` looks awesome, but to take it to perfection, kindly adjust the line under `Home` to look like the design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://github.com/donice/Portfolio-Microverse/assets/102612396/85c475cf-6426-4304-b379-8d72c7121c11)| ![image](https://github.com/donice/Portfolio-Microverse/assets/102612396/dc9d1fed-87de-4cf3-a6e4-416cc97c58fc)|